### ISSUE TITLE
Moving to different sortable area on multiple vertical containers cause strange animation issue

### ISSUE THREAD

**Issue by mateuszkosmider** [clauderic/dnd-kit#384]
This issue you can easily find in the storybook  - https://5fc05e08a4a65d0021ae0bf2-lsezdewkhf.chromatic.com/?path=/story/presets-sortable-multiple-containers--vertical 

This is quite strange behavior and it couldn't be easy to spot the first time. So I recorded a short video to show it on the example:

https://user-images.githubusercontent.com/974368/126969760-cf9de809-a7cf-4a7e-82b5-7d6ae77bc557.mp4

(issue occurs on: A3, A4, B3; works fine for: A5, B1) 

So basically on a vertical sortable component when you move any item (but not the last one) to a different contain there is something weird with reordering/animation (?) - all elements are moved but at the same time dragged element is removed from the list.

When you drag the last element everything is working fine.

**Comment by clauderic** [comment]
Duplicate of https://github.com/clauderic/dnd-kit/issues/64
